style.css

```diff
@@ -0,0 +1,1724 @@
+  display: grid;
+  padding: 7px;
+  background: #f7f7f9;
+  grid-template-columns: 8vh auto auto;
+}
+
+.navbar a:hover {
+  color: #6070ff;
+}
+
+.navbar a:active {
+  color: #2230d2;
+}
+
+/* This is logo  */
+.navbar .left {
+  text-align: center;
+  list-style: none;
+  flex-basis: 100px;
+  background-color: #fff;
+}
+
+/* This is the icon area */
+
+.navbar .right {
+  list-style: none;
+  flex-basis: 40px;
+  background-color: #fff;
+}
+
+.material-symbols-outlined {
+  background-color: #fff;
+}
+
+/* Add bacgorund and also do flex as column wise */
+.main-text {
+  height: 100%;
+  background-color: #fff;
+  background-image: url(./Microverse-Student-Project-1/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 0 0 0 56px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 21px 0 21px;
+  gap: 7px;
+}
+
+.main-header h1 {
+  color: black;
+  font-family: 'Crete Round', serif;
+  background-color: #fff;
+}
+
+/* Add the des which has flex that is flex grow, flex shrink, and flex basic */
+.main-header {
+  flex: 0 1 97px;
+  font-size: 29px;
+  font-weight: 400;
+  letter-spacing: 2.5px;
+}
+
+.dis {
+  align-self: center;
+}
+
+.dis p {
+  color: black;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.3;
+  background-color: #fff;
+}
+
+.connect h3 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+.icon {
+  background-color: #fff;
+  color: #fff;
+}
+
+.icon img {
+  background-color: #fff;
+}
+
+.social-media {
+  display: flex;
+  flex-wrap: nowrap;
+  background-color: #fff;
+}
+
+.social-media-icon {
+  background-color: #fff;
+  color: white;
+  display: flex;
+}
+
+.social-media-icon img {
+  margin: 17px;
+}
+
+.whole-main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Main contaniner of card Showing the total space all four required and spacing between them that is space-evenly */
+.contaniner {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: space-evenly;
+  padding: 16px;
+  width: 375px;
+  height: 2788px;
+  background-color: #e5e5e5;
+}
+
+/* The space that the card is taking  */
+.card {
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding: 8px;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  border-radius: 16px;
+  margin: 0 12px;
+}
+
+/* Center the image */
+.img {
+  display: grid;
+  justify-content: center;
+  border-radius: 12px;
+}
+
+.img-1 {
+  display: block;
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+.img-2 {
+  display: block;
+
+  /* background-image: url(./Pictures/Snapshoot\ Portfolio-2.png); */
+  width: 295px;
+  height: 220px;
+  margin-left: 14px;
+}
+
+.img-3 {
+  display: block;
+
+  /* background-image: url(./Pictures/Snapshoot\ Portfolio-3.png); */
```

Comment:
- [ ] Please remove this commented code, this will make your project not messy. 👍